segmentstore/storage/impl/src/main/java/io/pravega/segmentstore/storage/impl/rocksdb/RocksDBCache.java

```diff
@@ -95,6 +98,7 @@
         this.writeBufferSizeMB = config.getWriteBufferSizeMB() / MAX_WRITE_BUFFER_NUMBER;
         this.readCacheSizeMB = config.getReadCacheSizeMB();
         this.cacheBlockSizeKB = config.getCacheBlockSizeKB();
+        this.rocksDBLogLevel = config.getRocksDBLogLevel();
```

Comment:
I would suggest to add a `Statistics` attribute here and pass it to the db options, so we can access to the statistics of RocksDB (latency histograms, hits/misses, etc.). In debug mode, having this info periodically printed in the logs may be useful (e.g., every 5 minutes). See https://github.com/facebook/rocksdb/wiki/Statistics.